src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/HomeOfficeInstruct.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+public class HomeOfficeInstruct {
+
+    @JsonProperty("consumerReference")
+    private ConsumerReference consumerReference;
```

Comment:
jackson usually automatically converts these since the names are the same.  So you don't have to explicitly have to use an annotation.

Reply:
Since I removed SnakeCase, I removed these too